### DISCUSSION TITLE
Does NextJs have an official chat on Gitter or Discord

### DISCUSSION THREAD

**Discussion by Sushmeet** [vercel/next.js#14919] [Help] [answered]
Was looking to see if nextjs had an official chat on Discord. I have often found communities and chats in Discord to be quite helpful and welcoming to new members and it also serves as a great place to ask questions. 

I looked around on both gitter and discord and didn't find anything so was wondering if that could be something worth considering

Thanks for the awesome work that is being done here.

**Comment by cristiand391** [comment]
Not that I'm aware of, there was a [Spectrum's Next.js community](https://spectrum.chat/next-js) but it was closed some time ago in favor of GitHub Discussions.

**Reply by Sushmeet** [reply]
thanks, would there be any interest in creating perhaps a discord community channel. I personally feel they github discussions and discord both bring different sets of experiences and values to the audience.

**Reply by ItsWendell** [reply]
There's a unoffical NextJS channel on Reactiflux: https://www.reactiflux.com/

**Reply by Sushmeet** [reply]
Thank you @ItsWendell , this channel looks great. Thanks @cristiand391 also I think i have enough for now.

**Comment by cobrexus** [comment, marked as answer]
Vercel now does have a brand new Discord server just for Next.js: http://nextjs.org/discord

**Reply by stickyburn** [reply]
unable to accept invite? Is this on my end or the invitation itself? I'm new to Next and have a basic yet fundamental question to ask if possible :)

**Reply by cobrexus** [reply]
@yashank09 the links works for me, what happens when you click on it?

**Reply by stickyburn** [reply]
Maybe it was my PC, but link worked from my phone :) Thankyou!

**Reply by Industrial** [reply]
Hi. I get this:

![image](https://i.imgur.com/432pwq1.png)

**Reply by cobrexus** [reply]
@Industrial

1. Go to [discord.com/app](https://discord.com/app)
2. Click the <kbd>+</kbd> button at the bottom of the leftmost sidebar
3. Click <kbd>Join a Server</kbd> beneath **Have an invite already?**
4. Enter `nextjs` and click <kbd>Join Server</kbd>

Does that work for you?

**Comment by Syammed2429** [comment]
Hey guys can anyone help me how to make static.fun website please 
I mean if any user register on my website I want to give him his specific domain with his username. Same as linktree and static.fun website 

i need tutorial on it please help me
